target/linux/ath79/image/generic.mk

```diff
@@ -642,6 +642,19 @@ define Device/compex_wpj531-16m
 endef
 TARGET_DEVICES += compex_wpj531-16m
 
+define Device/compex_wpj558-16m
+  SOC := qca9558
+  IMAGE_SIZE := 16128k
+  DEVICE_VENDOR := Compex
+  DEVICE_MODEL := WPJ558
+  DEVICE_VARIANT := 16M
+  SUPPORTED_DEVICES += wpj558
```

Comment:
Is the vendor firmware OpenWrt or why do you need this?

Reply:
Ah, so this was supported with ar71xx. Somebody could have told me earlier ...
In this case, SUPPORTED_DEVICES is correct.